addons/binding/org.openhab.binding.gardena/src/main/java/org/openhab/binding/gardena/internal/GardenaSmartImpl.java

```diff
@@ -365,6 +381,14 @@ public void sendCommand(Device device, GardenaSmartCommandName commandName, Obje
         }
     }
 
+    private Integer getIntegerValue(Object value) {
+        try {
+            return Integer.valueOf(ObjectUtils.toString(value));
+        } catch (Exception ex) {
+            return null;
```

Comment:
Given it seems you always want to return an exception if we get here, it might make sense to return the exception here, then your control flow (near line 366) is not interrupted